servers/navigation_server_3d.h

```diff
@@ -304,7 +304,10 @@ class NavigationServer3D : public Object {
 	virtual void init() = 0;
 	virtual void sync() = 0;
 	virtual void finish() = 0;
-	virtual void free(RID p_object) = 0;
+	virtual void free_rid(RID p_rid) = 0;
+	[[deprecated]] virtual void free(RID p_rid) {
```

Comment:
IIRC usually we wrap these things in `ifndef DISABLE_DEPRECATED` in addition. 
Not sure if that's needed if we have `[[deprecated]]` also but so far we've been doing it.